.stylelintrc.json

```
@@ -246,7 +246,7 @@
     "scss/dollar-variable-pattern": "^([a-z][a-z0-9]*)(-[a-z0-9]+)*$",
     "scss/load-partial-extension": "never",
     "scss/no-duplicate-mixins": true,
-    "custom/require-nested-after-include": true
+    "@apostrophecms/require-nested-after-include": true
```

```suggestion
    "@apostrophecms/stylelint-no-mixed-decls": true
```

Could be overridden in project with mixed that are considered as "safe" (ie.  that do not contain nested declarations):

```js
"@apostrophecms/stylelint-no-mixed-decls": [
      true,
      {
        safeMixins: [
          "my-safe-mixin",
          "another-safe-mixin"
        ]
      }
    ]
```